server_admin/topics/threat/csrf.adoc

```diff
@@ -1,18 +1,10 @@
 
 === CSRF Attacks
 
-Cross-site request forgery (CSRF) is a web-based attack whereby HTTP requests are transmitted from a user that the
-web site trusts or has authenticated with(e.g. via HTTP redirects or HTML forms).  Any site that uses cookie based authentication is vulnerable to these types of attacks.
-These attacks are mitigated by matching a state cookie against a posted form or query parameter.
+A Cross-site request forgery (CSRF) attack uses HTTP requests from users that websites have already authenticated. Any site using cookie-based authentication is vulnerable to CSRF attacks. You can mitigate these attacks by matching a state cookie against a posted form or query parameter.
 
-The OAuth 2.0 login specification requires that a state cookie be used and matched against a transmitted state parameter.
-{project_name} fully implements this part of the specification so all logins are protected.
+The OAuth 2.0 login specification requires that a state cookie matches against a transmitted state parameter. {project_name} fully implements this part of the specification, so all logins are protected.
 
-The {project_name} Admin Console is a pure JavaScript/HTML5 application that makes REST calls to the backend {project_name} admin REST API.
-These calls all require bearer token authentication and are made via JavaScript Ajax calls.
-CSRF does not apply here.
-The admin REST API can also be configured to validate the CORS origins as well.
+The {project_name} Admin Console is a JavaScript/HTML5 application that makes REST calls to the backend {project_name} admin REST API. These calls all require bearer token authentication and consist of JavaScript Ajax calls, so CSRF is impossible. The admin can also configure the REST API to validate the CORS origins.
```

Comment:
It's officially referred to as the "admin REST API"  
best to say "You can configure the admin REST API..."